int32 class_id
string class_name
float32 confidence
UserRect bbox
geometry_msgs/Point center